pkg/driver/luks_util.go

```diff
@@ -86,10 +86,22 @@ func IsLuksMapping(exec k8sExec.Interface, devicePath string) (bool, string, err
 	return false, "", nil
 }
 
-func LuksResize(exec k8sExec.Interface, deviceName string) error {
+func LuksResize(exec k8sExec.Interface, deviceName string, passphrase string) error {
 	cryptsetupArgs := []string{"--batch-mode", "resize", deviceName}
```

Comment:
// Define arguments for cryptsetup
cryptsetupArgs := []string{"--batch-mode", "resize", deviceName}